creator-node/src/serviceRegistry.js

```diff
@@ -64,9 +63,14 @@ const initAudiusLibs = async () => {
   )
   const dataWeb3 = await AudiusLibs.Utils.configureWeb3(
     config.get('dataProviderUrl'),
-    null,
-    false
+    /* chainNetworkId */ null,
```

Comment:
will this work without the delegate pKey?

Reply:
it won't, but leaving as is for now